complex.c

```diff
@@ -1850,16 +1853,11 @@ nucomp_to_r(VALUE self)
 
 /*
  * call-seq:
- *    cmp.rationalize([eps])  ->  rational
+ *   rationalize(epsilon = nil) -> rational
  *
- * Returns the value as a rational if possible (the imaginary part
- * should be exactly zero).
+ * Equivalent to Complex#to_r.
  *
- *    Complex(1.0/3, 0).rationalize  #=> (1/3)
- *    Complex(1, 0.0).rationalize    # RangeError
- *    Complex(1, 2).rationalize      # RangeError
- *
- * See to_r.
+ * (Argument +epsilon+ is ignored.)
```

Comment:
`epsilon` is not ignored. It's called with `self.real.rationalize(epsilon)`. For example:

```ruby
irb(main):004> Complex(1.3333, 0).rationalize
=> (13333/10000)
irb(main):005> Complex(1.3333, 0).rationalize(0.1)
=> (4/3)
```

Reply:
Revised.